### PR TITLE
rbd-nbd: bug fix when running in container

### PR DESCRIPTION
**Pull request by dragonylffly**

### CODE REVIEW

File: src/tools/rbd_nbd/rbd-nbd.cc [hunk 1]
```diff
@@ -605,12 +605,27 @@ static int do_map(int argc, const char *argv[], Config *cfg)
   if (cfg->devpath.empty()) {
     char dev[64];
     bool try_load_module = true;
+    const char *path = "/sys/module/nbd/parameters/nbds_max";
```
Comment (amitkumar50) [inline src/tools/rbd_nbd/rbd-nbd.cc]: do we configure this path in ceph?

Comment (dragonylffly) [reply src/tools/rbd_nbd/rbd-nbd.cc]: No, it is a system path

**Comment by dillaman** [comment]
@dragonylffly Can you please create a tracker ticket and associate it w/ your commit message (via "Fixes: <url>")? This seems like a bug that should be backported to jewel and luminous.

**Comment by dragonylffly** [comment]
@dillaman updated

**Pull request merged**